_posts/2024-06-01-Announcing-Inbound-IPv6-support.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Announcing inbound IPv6 support in public preview"
+author_name: "Mads Damgård"
+toc: true
+toc_sticky: true
+---
+
+I am happy to announce the first part of our IPv6 implementation in App Service. Public preview of inbound IPv6 support for multi-tenant apps on Premium SKUs, Functions Consumption, Functions Elastic Premium, and Logic Apps Standard). We'll be adding IPv6 support in four stages.
+
+1. IPv6 inbound support (multi-tenant)
+1. IPv6 non-vnet outbound support (multi-tenant)
+1. IPv6 vnet outbound support (multi-tenant and App Service Environment v3)
+1. IPv6 vnet inbound support (App Service Environment v3 - both internal and external)
+
+Limitations in public preview:
+
+* Only a subset of regions are supported - see the list below.
+* Basic and Standard tier is currently not supported.
+* Functions Consumption may temporarily have extra IP addresses in the DNS result.
+* Functions Consumption and Elastic Premium may not remove the IPv4 address in IPv6 mode.
+* The IPv6 address is not visible in the API.
+* IP-SSL IPv6 bindings are not supported.
+
+## How does it work
+
+IPv6 inbound requires two things. An IPv6 address that accepts traffic coming in, and a DNS record that returns an IPv6 (AAAA) record. Finally you'll also need a client that can send and receive IPv6 traffic. This means that you may not be able to test it from you local machine since many networks today only support IPv4.
```

Comment:
```suggestion
IPv6 inbound requires two things. An IPv6 address that accepts traffic coming in, and a DNS record that returns an IPv6 (AAAA) record. Finally you'll also need a client that can send and receive IPv6 traffic. This means that you may not be able to test it from your local machine since many networks today only support IPv4.
```